### PR TITLE
add syslog module to realize the log forward

### LINKED ISSUE #1411: [FVT]Failed to find expected diskless logs in MN for RH6.8 ppc64
Using latest 27 Jun. daily build to install CN diskless RH6.8 on ppc64, then failed to find below logs on MN: ar/log/xcat/computes.log  according to design documentation. Neither xcatdebugmode=1 or 2 works.

Excepted logs:
```
Y3(xcatdebugmode=1 or xcatdebugmode=2) means the installation logs can be forwarded to /var/log/xcat/computes.log file on MN for redhat and sles.

running xcatroot....
Extracting root filesystem:
Done....
```
Failed to find them:
```
RUN:cat /var/log/xcat/computes.log | grep "running xcatroot...."

[cat /var/log/xcat/computes.log | grep "running xcatroot...."] Running Time:0 sec
RETURN: rc = 1
OUTPUT:
CHECK:output =~ running xcatroot....    [Failed]

RUN:cat /var/log/xcat/computes.log | grep "Extracting root filesystem"

[cat /var/log/xcat/computes.log | grep "Extracting root filesystem"] Running Time:0 sec
RETURN: rc = 1
OUTPUT:

RUN:cat /var/log/xcat/computes.log | grep "Done...."

[cat /var/log/xcat/computes.log | grep "Done...."] Running Time:0 sec
RETURN: rc = 1
OUTPUT:
```

### PR DESCRIPTION
**Pull request by penguhyang**
fix #1411

### CODE REVIEW

File: xCAT-server/share/xcat/netboot/rh/dracut_033/xcatroot [hunk 1]
```diff
@@ -1,244 +1,236 @@
...
 
-
 [ "$xcatdebugmode" > "0" ] && logger -t xcat -p debug "running xcatroot...."
 [ "$xcatdebugmode" > "0" ] && logger -t xcat -p debug "MASTER=$MASTER XCATIPORT=$XCATIPORT"
 
-
-if [ $NODESTATUS -ne 0 ];then
-[ "$xcatdebugmode" > "0" ] && logger -t xcat -p debug "nodestatus: netbooting,reporting..."
-/tmp/updateflag $MASTER $XCATIPORT "installstatus netbooting"
+if [ $NODESTATUS -ne 0 ]; then
```
Comment (immarvin) [inline xCAT-server/share/xcat/netboot/rh/dracut_033/xcatroot]: `` "$NODESTATUS" != "0"``

Comment (penguhyang) [inline xCAT-server/share/xcat/netboot/rh/dracut_033/xcatroot]: fixed.

File: xCAT-server/share/xcat/netboot/rh/dracut/xcatroot [hunk 2]
```diff
@@ -16,18 +16,25 @@ if [ $? -ne 0 ]; then
...
+    [ "$xcatdebugmode" > "0" ] && logger -t xcat -p debug "nodestatus: netbooting,reporting..."
+    /tmp/updateflag $MASTER $XCATIPORT "installstatus netbooting"
 fi
 
 if [ ! -z "$imgurl" ]; then
 	if [ xhttp = x${imgurl%%:*} ]; then
 		NFS=0
 		FILENAME=${imgurl##*/}
 		while [ ! -r "$FILENAME" ]; do
+            [ "$xcatdebugmode" > "0" ] && logger -t xcat -p debug "downloading $imgurl...."
```
Comment (immarvin) [inline xCAT-server/share/xcat/netboot/rh/dracut/xcatroot]: the indent is not correct

**Comment by immarvin** [comment]
i am ok with the PR, agree to merge

**Pull request merged**